release-notes.md

```diff
@@ -20,6 +20,9 @@ The following components are new in Tanzu Application Platform v0.4.0:
 
 The following components have been updated in Tanzu Application Platform v0.4.0:
 
+- Supply Chain Security Tools for VMware Tanzu
```

Comment:
To be consistent `use Supply Chain Security Tools` as you removed `VMware Tanzu`